test/e2e/page/table-editable-page-object.ts

```diff
@@ -43,7 +43,6 @@ export default class TableEditablePageObject extends TableFilteringPageObject {
     const bodyCell = this.tableWrapper.findBodyCell(row, 4);
     const input = bodyCell.findFormField().find('input');
     await this.click(bodyCell.toSelector());
```

Comment:
When the editable cell is clicked the input is auto-focused. A second click to the input is unnecessary and sometimes leads to the text cursor position change: it moves from behind the last symbol to the middle of the visible input area. That breaks the input cleanup code from below that relies on the cursor position.